readme.md

```diff
@@ -33,7 +33,107 @@ npm install colette --save
 
 ---
 ## Download
-- See the [`dist/`](https://github.com/20minutes/colette/dist/) 
+- See the [`dist/`](https://github.com/20minutes/colette/dist/)
 directory for ready-to-use css files.
-- If you want to use Colette as a kickstarter for custom stylus/css 
+- If you want to use Colette as a kickstarter for custom stylus/css
 projects, check [`src/`](https://github.com/20minutes/colette/assets/).
+
+---
+## Use gulp
+
+To compile your stylus, use `gulp` and `gulp-stylus`
+
+```javascript
+/* gulpfile.js */
+gulp.task('css', function () {
+   return gulp.src(cfg.cssDir + '*.styl')
+   .pipe(stylus({
+        include: [
+            'node_modules',
```

Comment:
If colette is downloaded (instead of bower or npm installed), then the path to node_modules would not be relevant here.
1. comment this line, and add a mention to uncomment if colette has been npm installed:
 ````
// uncomment this line if you have installed colette via npm
// 'node_modules',
````
2. Maybe add an example (commented as well) for bower?